chatwoot/chatwoot
# feat: Replace `attributeMixin` within the component

# Pull Request Template

## Description

This PR will replace the usage of `attributeMixin` within the component itself. And moved the component from option API to composition API and updated the styles and related component

Fixes https://linear.app/chatwoot/issue/CW-3444/rewrite-attributemixin-mixin-to-a-composable

## Type of change


## How Has This Been Tested?

**Test cases**
1. Add custom attributes for both conversation and contact from the settings
2. See all attributes are showing based on the conversation and contact in both conversation and contact sidebar.
3. Try all CRUD operations like EDIT, DELETE. 
4. Check whether styles are properly showing or not (Background color based on odd/even)

## Checklist:
